content/en/news/releases/1.5.x/announcing-1.5/change-notes/index.md

```diff
@@ -4,4 +4,79 @@ description: Istio 1.5 release notes.
 weight: 10
 ---
 
-TBD
+## Traffic management
+
+- **Improved** performance of `ServiceEntry` updates #19305.
+- **Improved** Envoy sidecar readiness probe to more accurate determine readiness #18164.
```

Comment:
I can't make sense of this one, @howardjohn - suggestions?  Seems a bug fix we don't need to mention?